docs/LOCAL_DEPLOYMENT.md

```diff
@@ -19,7 +19,14 @@ The easiest way to run this accelerator is in a VS Code Dev Containers, which wi
 
 > NOTE: It may take up to an hour for the application to be fully deployed. If you see a "Python Developer" welcome screen or an error page, then wait a bit and refresh the page.
 
-**Notes:** the default auth type uses keys, if you want to switch to rbac, please run `azd env set AUTH_TYPE rbac`.
+> NOTE: The default auth type uses keys that are stored in the Azure Keyvault. If you want to use RBAC-based auth (more secure), please run before deploying:
+
+```bash
+azd env set AZURE_AUTH_TYPE rbac
```

Comment:
Could someone confirm that this is the correct way to set up? What was previously in this file definitely didn't work, but not sure if this is the best way to do this set up?

Reply:
Looks good 👍